src/utils/spreadsheet.ts

```diff
@@ -0,0 +1,17 @@
+import { Nullable, Sheet, Spreadsheet } from 'models';
+
+const getSheetByTitle = (spreadsheet: Spreadsheet, title: string): Nullable<Sheet> => {
+  if (!spreadsheet || !spreadsheet.sheets) {
+    return null;
+  }
+
+  for (const sheet of spreadsheet.sheets) {
```

Comment:
🤷‍♂ This can be simplified to 
```suggestion
  return spreadsheet.sheets.find((sheet: Spreadsheet) => title === sheet.title);
```